.gitlab-ci.yml

```diff
@@ -954,6 +954,9 @@ test_full_integration:
 
 release_docker_images:
   when: manual
+  only:
+    variables:
+      - $PUBLISH_ARTIFACTS == "true"
```

Comment:
Can't we just get rid of this variable if the stage is manual anyway?

Reply:
Yes, we could and that was my original intention. But then I thought that it is rather adding noise to the Pipelines on our daily work and might create confusion among the team.

Previously that we needed to publish master "once on a while" it could make sense, but now it will strictly used only for releases.

Unless you strongly oppose to it, I'd like to keep it clean for the daily use and a bit more complex for you and me on releases time.

Reply:
:+1: